cachito/web/static/api_v1.yaml

```diff
@@ -277,6 +277,75 @@ paths:
                 $ref: "#/components/schemas/RequestConfiguration"
       security:
       - "Kerberos Authentication": []
+  "/requests/{id}/content-manifest":
+    get:
+      description: Return the content manifest of a request
+      parameters:
+      - name: id
+        in: path
+        required: true
+        description: The ID of the Cachito request to retrieve the content manifest for
+        schema:
+          type: integer
+      responses:
+        "200":
+          description: The content manifest of the Cachito request
+          content:
+            application/json:
+              schema:
+                type: object
+                nullable: true
```

Comment:
Is this actually true?

Reply:
According to the original schema, it is. We could remove it here since we are not returning null though, since this is just for documenting what cachito is returning.

Reply:
Removed